B/blis/build_tarballs.jl

```diff
@@ -54,17 +59,30 @@ case ${target} in
         # Use Cortex-A57 for the moment.
         export BLI_CONFIG=cortexa57
         export BLI_THREAD=openmp
+        export BLI_F77TYPE=64
         ;;
     *"arm"*"linux"*) 
         export BLI_CONFIG=cortexa9
         export BLI_THREAD=none
+        # Keep 32-bit BLAS interface for 32-bit processors.
+        export BLI_F77TYPE=32
         ;;
     *)
+        # Default (Generic) configuration without optimized kernel.
+        export BLI_CONFIG=generic
+        export BLI_THREAD=none
+        export BLI_F77TYPE=64
         ;; 
 
 esac
 
-./configure -p ${prefix} -t ${BLI_THREAD} ${BLI_CONFIG}
+# For 64-bit builds, add _64 suffix to exported BLAS routines.
+# This corresponds to ILP64 handling of OpenBLAS thus Julia.
+if [ ${BLI_F77TYPE} = 64 ]; then
```

Comment:
Can simply use [`${nbits}`](https://docs.binarybuilder.org/dev/build_tips/#Automatic-environment-variables) instead of setting `BLI_F77TYPE`?

Reply:
Well, I just wanted to note that this 64-bit/32-bit switch is effective only for the F77 BLAS interface.

Integer type used inside BLIS is in fact controlled by yet another options.

Maybe `BLI_F77BITS` would be the best?

Reply:
Or set
```sh
export BLI_F77BITS=${nbits}
```
just once?

Reply:
OK! I'll update.

Reply:
Probably I wasn't clear enough (but I hoped the link to the documentation would help), but `nbits` is _already_ defined, so you don't have to do anything.

Reply:
Oh sorry! I'll edit.